docs/general/manage-tokens/README.mdx

```diff
@@ -128,29 +151,13 @@ schemes and compatible wallets.
 
 | Consensus or ParaTime | Address Format | Digital Signature Scheme    | Supported Wallets                                           |
 |-----------------------|----------------|-----------------------------|-------------------------------------------------------------|
-| Consensus             | `oasis1`       | ed25519                     | <ul><li>Oasis Wallet - Web</li><li>Oasis Wallet - Browser Extension</li><li>Oasis CLI</li></ul> |
-| Sapphire              | `0x`, `oasis1` | secp256k1, ed25519, sr25519 | <ul><li>Metamask and other EVM-compatible wallets (transfers only)</li><li>Oasis Wallet - Browser Extension</li><li>Oasis Wallet - Web (deposits and withdrawals only)</li><li>Oasis CLI</li></ul> |
+| Consensus             | `oasis1`       | ed25519                     | <ul><li>ROSE Wallet - Web</li><li>ROSE Wallet - Browser Extension</li><li>Oasis CLI</li></ul> |
+| Sapphire              | `0x`, `oasis1` | secp256k1, ed25519, sr25519 | <ul><li>Metamask and other EVM-compatible wallets (transfers only)</li><li>ROSE Wallet - Browser Extension</li><li>ROSE Wallet - Web (deposits and withdrawals only)</li><li>Oasis CLI</li></ul> |
 | Cipher                | `oasis1`       | secp256k1, ed25519, sr25519 | <ul><li>Oasis CLI</li></ul> |
-| Emerald               | `0x`, `oasis1` | secp256k1, ed25519, sr25519 | <ul><li>Metamask and other EVM-compatible wallets (transfers only)</li><li>Oasis Wallet - Browser Extension</li><li>Oasis Wallet - Web (deposits and withdrawals only)</li><li>Oasis CLI</li></ul> |
+| Emerald               | `0x`, `oasis1` | secp256k1, ed25519, sr25519 | <ul><li>Metamask and other EVM-compatible wallets (transfers only)</li><li>ROSE Wallet - Browser Extension</li><li>ROSE Wallet - Web (deposits and withdrawals only)</li><li>ROSE CLI</li></ul> |
```

Comment:
I really don't like "ROSE CLI" :shrug:

Reply:
Yeah please change the CLI back.

Reply:
Oops, where did ROSE CLI come from lol ;)